test-configs/operator-wandb/fmb.yaml

```diff
@@ -49,6 +50,9 @@ app:
     requests:
       cpu: "100m"
       memory: "128Mi"
+  image: # TODO: DO NOT SUBMIT - remove this and all image overrides
```

Comment:
do we want to override this here? or is it easier to this in the channel?

Reply:
Also, these files are just used for running tests and local dev, so they will have no bearing on the images used anywhere else.

Reply:
sounds good, I'll plan to remove this once I'm done with local testing before commit